sds_system/accounts/templates/accounts/base.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>SDS System</title>
+</head>
+<body>
+     {% block content %}
+     <!--Content Goes Here-->
+    {% endblock content%}
+</body>
+</html>
```

Comment:
Its better to keep the templates at the project level outside the app modules. So that, we don't have to define templates folder for every single app.
i.e: 
sds_system/accounts/
sds_system/templates/accounts/base.html